pkg/pillar/cmd/nim/nim.go

```diff
@@ -885,8 +944,26 @@ func (n *nim) ingestDevicePortConfigFile(oldDirname string, newDirname string, n
 	}
 }
 
-func (n *nim) updateLastResortDPC(reason string) {
-	n.Log.Functionf("updateLastResortDPC")
+func (n *nim) toggleLastResortDPC() {
```

Comment:
To me the name "toogle" means to always change it (from on to off or from off to on).
"maybeToggle", or "reevaluate" or "deternine" do not have that connotation.

Also, it would be good to have a coment for the function indicating that it updated enabledLastResort based on the result.

Reply:
Renamed to `reevaluateLastResortDPC` and added a comment.